libkbfs/folder_block_manager.go

```diff
@@ -1101,7 +1110,10 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 	}
 	if len(ptrs) == 0 && !shortened {
 		complete = true
-		return nil
+
+		// Add a new gcOp to show other clients that they don't need
+		// to explore this range again.
+		return fbm.finalizeReclamation(ctx, nil, nil, latestRev)
```

Comment:
> Ideally, though, we'd have 104 know to mark 75-104, right? I guess that might turn out to be complicated in edge cases where two clients are QRing at the same time.

Yeah, exactly, the races are a little tough to think through.  Maybe `folderBranchOps.finalizeGCOp` can update the `LatestRev` if the revision number is sequential.  But yeah, we can fix that up later.

> Do I understand it correctly?

Yeah.  In practice those QR sequences can be 20-25 long for each TLF, and they are fetched after every restart.  Especially when they are mdv2 revisions, this causes people to hit throttling whenever they restart KBFS and load all their chat-inbox-related TLFs.  So what I'm trying to do here is make sure we only have to fetch the latest MD for each TLF, rather than a range of arbitrary length.